packages/ui/src/store/constant.js

```diff
@@ -4,10 +4,7 @@ export const drawerWidth = 260
 export const appDrawerWidth = 320
 export const headerHeight = 80
 export const maxScroll = 100000
-export const baseURL =
-    import.meta.env.PROD === true
-        ? window.location.origin
-        : window.location.origin.replace(`:${import.meta.env.VITE_PORT ?? '8080'}`, ':3000')
+export const baseURL = window.location.origin
 export const uiBaseURL = window.location.origin
```

Comment:
@mokeyish just noticed there are 2 urls:
`baseURL` and `uiBaseURL` with same value `window.location.origin`, do we need both?

Also when running `pnpm dev` on `localhost:8080`, the `baseURL` becomes `localhost:8080`. This makes the `ChatMessage.jsx` not able to receive streaming because the server is `localhost:3000` - https://github.com/FlowiseAI/Flowise/blob/main/packages/ui/src/views/chatmessage/ChatMessage.jsx#L609

Reply:
This may be because vite does not support proxy SSE or because it only proxy the path of /api/?

Reply:
perhaps it doenst support socket proxy. we probably need to get the server port and replace it `const serverPort = parseInt(dotenv.config({ processEnv: {}, path: '../server/.env' }).parsed?.['PORT'])`

Reply:
I think the possible reason is that vite only proxy the `api/` prefix.


<img width="840" alt="图片" src="https://github.com/FlowiseAI/Flowise/assets/16131917/281fa584-834f-4cbe-976a-b7a41aca6e7d">

Reply:
@mokeyish do you have any fix for that?

Reply:
No, I'm on a business trip recently, so it's not convenient.

Reply:
Fix by #2396